yarn.lock

```diff
@@ -278,6 +278,16 @@
     lodash "^4.17.4"
     lodash-decorators "^4.3.5"
 
+"@shopify/react-html@^8.0.10":
```

Comment:
Why is this updated?

Reply:
Since quilt is a mono-repo where some packages pull in `react-html`, the build I ran pulled in the latest `8.0.x` version that wasn't available when `@shopify/dates@0.1.23` was released. This has no impact on `@shopify/dates` since it doesn't touch `react-html`

Reply:
_It really doesn't matter_ that I checked these in but I like to be transparent for questions like this. I'm glad you asked!

Reply:
thanks for explaining :)